src/main/java/no/unit/nva/doi/FetchDoiMetadata.java

```diff
@@ -90,19 +90,22 @@ public GatewayResponse handleRequest(Map<String, Object> input, Context context)
             System.out.println(e.getMessage());
             logger.log(e.getMessage());
             return errorGatewayResponse(e.getMessage(), INTERNAL_SERVER_ERROR.getStatusCode());
-
         }
     }
 
+    private Map<String, String> addContentLocationToHeaders(MetadataAndContentLocation doiMetadata) {
```

Comment:
Would the other way be bette? metadataAndContentLocation.addToHeaders(Map<String,String> headers)

Reply:
`addContentLocationToHeaders()` is a pure function. addToHeaders does the same with a side-effect.

Reply:
renamed and refactored the method